app_feup/lib/model/entities/time_utilities.dart

```diff
@@ -0,0 +1,7 @@
+extension TimeString on DateTime {
+  String toTimeString() {
```

Comment:
`toTimeString` may be a bit of a too generic name since this can take into account other time related information, such as seconds or even the current date. How about calling it `toTimeAndHourString`?

Reply:
Done! I called it `toTimeHourMinString`, so that we make sure what we will get when using this method. For example, we won't get information on seconds.